ocaml/xapi/xapi_globs.ml

```diff
@@ -1501,6 +1503,10 @@ module Resources = struct
       , "Executed to clear certain UEFI variables during clone"
       )
     ; ("varstore_dir", varstore_dir, "Path to local varstored directory")
+    ; ( "fallback_auth_dir"
+      , fallback_auth_dir
+      , "Fallback path for auth files installed by RPMs"
```

Comment:
It's not mandatory that the files come from RPMs so I wouldn't mention them here.